cli/internal/context/context.go

```diff
@@ -154,12 +156,9 @@ func WithGraph(repoRoot fs.AbsolutePath, rootPackageJSON *fs.PackageJSON, cacheD
 		// until all parsing is complete
 		parseJSONWaitGroup := &errgroup.Group{}
 		for _, workspace := range workspaces {
-			relativePkgPath, err := filepath.Rel(rootpath, workspace)
-			if err != nil {
-				return fmt.Errorf("non-nested package.json path %w", err)
```

Comment:
Note for readers: this error got moved into the function. It's currently impossible to reach this error case because of how the paths are generated. 🤷‍♂️